### ISSUE TITLE
Example stuck at get default allocator

### ISSUE THREAD

**Issue by Duallife** [micro-ROS/micro_ros_arduino#1073]
## Issue

- Hardware description: esp32 dev module
- Installation type: pre-compiled library, Arduino ide
- Version or commit hash: foxy

Sorry that might be a silly question, I'm a newbie to ROS

#### issue
So the library is installed through zip. The example is "micro-ros publisher". The program was successfully compiled. Though it keep going into error loop. The programme is stucked after "get default allocator".
Serial output get some output like this
![image](https://user-images.githubusercontent.com/85776119/178139528-c4f8c682-8339-4601-8047-625204745b08.png)

I'm running ubuntu on a virtual machine through virtual box

So basically what I do is install the library and compile it to esp32. Is that any step I'm missing, e.g. launching some service?
